pipenv/vendor/dotenv/main.py

```diff
@@ -94,6 +94,13 @@ def set_as_environment_variables(self, override=False):
         for k, v in self.dict().items():
             if k in os.environ and not override:
                 continue
+            # With Python 2 on Windows, ensuree environment variables are
+            # system strings to avoid "TypeError: environment can only contain
+            # strings" in Python's subprocess module.
+            if sys.version_info.major < 3 and sys.platform == 'win32':
+                from pipenv.utils import fs_str
```

Comment:
Is there any reason not to just do this all the time? Why not just always use `fs_str` here?